build/travis-debug/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Simulate our Maven build on Travis-CI
+# WORKDIR ends up as /home/travis/build
+# Source directory should be mounted here
+
+FROM quay.io/travisci/travis-jvm
```

Comment:
I have to ask about the security of this image. Who provides it? How is it vetted?